src/app/frontend/chrome/chrome.scss

```diff
@@ -84,3 +84,27 @@ a {
 [role='tabpanel'] {
   transition: none;
 }
+
+// Overrides wt-responsive-table directive styles. More details at
+// https://github.com/awerlang/angular-responsive-tables.
+@media only screen and (max-width: 800px) {
+  .responsive {
+    tr {
+      &:not(:last-child) {
```

Comment:
Not sure why, but it requires two empty lines if I'm putting it after `border: 0;`. That's why I've moved it to the beggining.